src/Compilers/CSharp/Portable/Symbols/MethodSymbol.cs

```diff
@@ -1153,12 +1153,25 @@ ImmutableArray<AttributeData> IMethodSymbol.GetReturnTypeAttributes()
         /// <summary>
         /// Build and add synthesized return type attributes for this method symbol.
         /// </summary>
-        internal virtual void AddSynthesizedReturnTypeAttributes(PEModuleBuilder moduleBuilder, ref ArrayBuilder<SynthesizedAttributeData> attributes)
+        internal void AddSynthesizedReturnTypeAttributes(PEModuleBuilder moduleBuilder, ref ArrayBuilder<SynthesizedAttributeData> attributes)
```

Comment:
>internal void AddSynthesizedReturnTypeAttributes(PEModuleBuilder moduleBuilder, ref ArrayBuilder<SynthesizedAttributeData> attributes) [](start = 8, length = 134)

We probably should override this for a PEMethodSymbol. None of the synthesized attributes should be added for it.